http/server.go

```diff
@@ -258,15 +264,29 @@ func (s *Server) HandleFunc(path string, f auth.AuthenticatedHandlerFunc) {
 	s.Router.HandleFunc(path, s.Auth.Wrap(f))
 }
 
-func (s *Server) loadExtraAssets(folder string) {
-	files, err := ioutil.ReadDir(folder)
+func (s *Server) loadExtraAssets(folder, prefix string) {
+	files := []string{}
+
+	err := filepath.Walk(folder, func(path string, info os.FileInfo, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+
+		if filepath.Ext(path) == ".go" {
```

Comment:
It seems that was required in the previous version of the patch. Maybe it can now be removed

Reply:
ok, I have removed this